change/@fluentui-react-motion-2679d27b-fa54-468d-9a6e-88febcf5eab0.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33431&commitId=321fa7bf7e29548a9a4fc50bcf7647bd941052b2&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Basic High Contrast.chromium.png|2| Changed|
|Avatar Converged.badgeMask.chromium.png|12| Changed|
|Avatar Converged.Badge Mask RTL.chromium.png|5| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|984| Changed|
|Drawer.overlay drawer full.chromium.png|2859| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>